### DISCUSSION TITLE
Accessing program level tags

### DISCUSSION THREAD

**Discussion by SteveZuercher** [ottowayi/pycomm3#136] [Q&A]
Hi, I am unable to find documentation on the proper syntax to access program level tags. I looked through the documentation on read the docs, then through the python program, but did not have any luck.

is it \<Program>.<tag>?

There is also a typo in the documentation. The last element should be [2,1], not [2,3].
<img width="381" alt="image" src="https://user-images.githubusercontent.com/43651516/116604621-fb34e880-a8f3-11eb-8d95-659a4b31613c.png">

Thank you for your time.

**Comment by ottowayi** [comment]
Program tags use the same syntax as you would in an HMI, it's `Program:<program_name>.<tag>`, e.g. `Program:MainProgram.SomeTag`.  

And yeah that is a typo, thanks for noticing it.  I will get it fixed as well as add something about program-scoped tags.

**Reply by ottowayi** [reply]
Docs should be updated now
